draft-bishop-httpbis-http2-additional-certs.md

```diff
@@ -552,9 +548,7 @@ stream error of type `PROTOCOL_ERROR`.
   0                   1                   2                   3
   0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
  +-------------------------------+-------------------------------+
- |        Request-ID (16)        |      Extension-Count (16)     |
- +-------------------------------+-------------------------------+
- |                          Extensions(?)                      ...
+ |        Request-ID (16)        |          Request (?)        ...
```

Comment:
Does this ID replicate the context that is inside the request?

Reply:
Also, it seems like this is in violation of the recommendation in TLS for unpredictable contexts.